src/main/java/frc/trigon/robot/subsystems/climber/ClimberVisualization.java

```diff
@@ -0,0 +1,163 @@
+package frc.trigon.robot.subsystems.climber;
+
+import edu.wpi.first.math.geometry.*;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import frc.trigon.robot.RobotContainer;
+import org.littletonrobotics.junction.Logger;
+
+/**
+ * A class that visualizes the climber in the simulation, using a mechanism and calculating the poses.
+ */
```

Comment:
this doesn't only visualize in simulation 
What is "calculating the poses" (mention ascope), what is a "mechanism" (replace with Mechanism2d)?